xbmc/music/tags/TagLoaderTagLib.h

```diff
@@ -49,6 +49,9 @@ class CTagLoaderTagLib : public MUSIC_INFO::IMusicInfoTagLoader
   static int POPMtoXBMC(int popm);
 
 template<typename T>
-   static bool ParseTag(T *tag, EmbeddedArt *art, MUSIC_INFO::CMusicInfoTag& infoTag);
+  static bool ParseTag(T* tag, EmbeddedArt* art, MUSIC_INFO::CMusicInfoTag& infoTag);
+
+private:
+  static int m_totalLen;
```

Comment:
If you insist that this needs to be a static member variable, then please follow the coding guidelines and use proper naming: https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#member-variables

Reply:
Done